content/homework.rst

```diff
@@ -123,8 +150,13 @@ You shouldn't be dependent on work for social interactions.
 Communication
 =============
 
-Use "away" and "busy" functionality of you instant messaging tool to let other
-know about your status.
+Use "away" and "busy" functionality of you instant messaging tool to let other know about your status.
+
+Use a dedicated account for work and log in with that account
+only when you are at work.
+On the same line, join the team IRC chat only when you are *at work*.
```

Comment:
true..... but for IRC away status is advertised only for private messages and most IRC clients will not constantly pool your whois status to see if you are away... but maybe they should... or we can switch to another protocol/tools... maybe Slack as it provides an IRC/XMMP gateway.

we will still use IRC as is the protocol used by open source projects and we want to become some day an open project

I have removed this part